Add vhost parameter

lancefrench opened the pull request:
I wanted to use the stress-tester to create bad actors against specific vhosts.  This pull request adds a `--vhost` parameter to allow using something other than the default.

Commit:
Fixes typo in password parameter usage note

Commit:
Adds vhost parameter

Adds vhost parameter per https://www.rabbitmq.com/uri-spec.html

Commit:
Adds documentation of vhost flag

Updates flag docs for vhost
Fixes typo in "executable"
Updates tab spacing for flag descriptions

agocs commented on main.go:
```diff
@@ -45,7 +46,8 @@ func main() {
 	porto := "amqp://"
-	uri := porto + c.String("user") + ":" + c.String("password") + "@" + c.String("server") + ":" + c.String("port")
+	uri := porto + c.String("user") + ":" + c.String("password") + "@" + c.String("server") +
+		":" + c.String("port") + "/" + c.String("vhost")
```
According to section 2.4 of the rabbitmq URI spec:

>The vhost component may be absent; this is indicated by the lack of a "/" character following the amqp_authority. An absent vhost component is not equivalent to an empty (i.e. zero-length) vhost name.

I'm concerned that adding the `/` character here might lead to some weird behavior in an edge case. It might be better to see if the vhost parameter starts with `/` and add the `/` if not, then concatenate that string onto the rest of the uri.

lancefrench replied:
Sounds good, I'll make the `/` conditional based on input later this evening.

agocs replied:
👍 👍 Thanks!

Commit:
Adds vhost parameter processing

Updates vhost logic to better comply with the URI spec.  The default value is a nil vhost.
